website/docs/getting-started/faq.md

```diff
@@ -7,36 +7,42 @@ sidebar_position: 4
 
 ## What kind of apps can be built on Appsmith?
 
-You can use Appsmith to build custom forms, dashboards & workflows. Tools like Customer Support Dashboards, Offer Management Consoles & Product Cataloging Systems are all prime use cases for Appsmith. You can also use it as a clean UI layer over manually triggered scripts, confusing deployment pipelines & basic database CRUD operations. There's very little preventing you from using Appsmith to bring any idea to life but we've optimized the product to make building internal tools a hassle-free experience.
+You can use Appsmith to build custom forms, dashboards & workflows. Tools like Customer Support Dashboards, Offer Management Consoles & Product Cataloging Systems are all prime use cases for Appsmith. You can also use it as a clean UI layer over manually triggered scripts, confusing deployment pipelines & basic database CRUD operations. There's very little preventing you from using Appsmith to bring any idea to life, but we've optimized the product to make building internal tools a hassle-free experience.
 
 ## Can I write code in Appsmith?
 
-Yes, absolutely. You can write JS code anywhere inside the product within `{{ }}`. We know how complex and intricate business logic can be and trying to build it without code can get excruciating. Appsmith bridges mundane API and UI configurations with powerful code expressions to let you build the apps the way you see them fit.
+Yes, absolutely. You can write JavaScript code anywhere inside the product within curly braces `{{ }}`, or using standard syntax within [JS Objects](/core-concepts/writing-code/javascript-editor-beta). We know how complex and intricate business logic can be, and trying to build it without code can get excruciating. Appsmith bridges mundane API and UI configurations with powerful code expressions to let you build the apps the way you see them fit.
 
-## What type of data security does appsmith provide?
+Read more about writing code [here](/core-concepts/writing-code).
```

Comment:
Could you please follow the style guide for links? (https://developers.google.com/style/link-text) and make the necessary changes.